ember-cli-build.js

```diff
@@ -138,6 +138,9 @@ module.exports = function(defaults) {
         'ember-resolver/utils/create':true,
         'ember-resolver/utils/make-dictionary':true,
         'ember-resolver/utils/module-registry':true
+      },
+      parseOptions: {
+        ecmaVersion: 8
```

Comment:
should we use year-base versions (e.g. 2017 for 9, 2018 for 11?) instead? it's more descriptive that 8, 9, etc..

Reply:
Good point, will make the change.